Model/Source/AgeCheckOptions.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+namespace MyParcelNL\Magento\Model\Source;
+
+use Magento\Eav\Model\Entity\Attribute\Source\AbstractSource;
+
+class AgeCheckOptions extends AbstractSource
+{
+    /**
+     * @return array
+     */
+    public function getOptionArray()
+    {
+        return [
```

Comment:
Is dit iets wat vaker in de code voorkomt?

Reply:
De mannier hoe dit geprogrammeerd is, dat komt vaker voor. 
De opties die er in staan, zijn specifiek voor de age check. Ik heb ook naar de andere producten gekeken en deze opties wordt op dit moment nergens anders gebruikt.